validator-generator/src/main/java/io/avaje/validation/generator/TypeReader.java

```diff
@@ -83,8 +83,9 @@ void read(TypeElement type) {
     }
 
     for (final FieldReader localField : localFields) {
-      allFields.add(localField);
-      allFieldMap.put(localField.fieldName(), localField);
+      if (allFieldMap.put(localField.fieldName(), localField) == null) {
```

Comment:
What is this about? Looks like a bug fix unrelated to package name fix?

Reply:
on JDT for whatever reason, I noticed the generated fields getting duplicated, traced it back to here

Reply:
I guess the fields get read twice somehow so there were duplicates

Reply:
Right. Noting that someone reading "Fix JDT Errors" isn't going to relate that to ... "Fix duplicate generated fields".